src/pages/projectSettings/tabs/GithubCommitQueueTab/transformers.ts

```diff
@@ -58,41 +68,41 @@ export const gqlToForm: GqlToFormFunction<FormState> = (
 
   return {
     github: {
-      prTestingEnabled: projectRef.prTestingEnabled,
-      manualPrTestingEnabled: projectRef.manualPrTestingEnabled,
+      prTestingEnabled,
+      manualPrTestingEnabled,
       prTesting: {
         githubPrAliasesOverride: override(githubPrAliases),
         githubPrAliases,
       },
       githubTriggerAliases,
-      githubChecksEnabled: projectRef.githubChecksEnabled,
+      githubChecksEnabled,
       githubChecks: {
         githubCheckAliasesOverride: override(githubCheckAliases),
         githubCheckAliases,
       },
-      gitTagVersionsEnabled: projectRef.gitTagVersionsEnabled,
+      gitTagVersionsEnabled,
       users: {
-        gitTagAuthorizedUsersOverride: override(
-          projectRef.gitTagAuthorizedUsers
-        ),
-        gitTagAuthorizedUsers: projectRef.gitTagAuthorizedUsers,
+        gitTagAuthorizedUsersOverride:
+          projectType !== ProjectType.AttachedProject ||
+          !!gitTagAuthorizedUsers,
```

Comment:
Update this logic to correctly allow users to override repo authorized users.

Reply:
Can you please explain this change a bit more?

Reply:
ah nvm, I think I understand